src/Integration.Vsix.UnitTests/SonarAnalyzerDeactivationManagerTests.cs

```diff
@@ -169,7 +178,7 @@ public void SonarAnalyzerCollisionManager_MultipleReferencesWithSameName_NonColl
 
         #region Helper AnalyzerReference class
 
-        private class TestAnalyzerReference : AnalyzerReference
+        internal class TestAnalyzerReference : AnalyzerReference
```

Comment:
If shared, might be better to extract to a separate helper class in a file of its own.